vnext/Shared/MemoryMappedBuffer.cpp

```diff
@@ -0,0 +1,154 @@
+#include "pch.h"
+#include "MemoryMappedBuffer.h"
+
+#include <werapi.h>
+#include <windows.h>
+
+namespace {
+
+class MemoryMappedBuffer : public facebook::jsi::Buffer {
+ public:
+  MemoryMappedBuffer(const wchar_t *const filename, uint32_t offset);
+
+  size_t size() const override;
+  const uint8_t *data() const override;
```

Comment:
>const uint8_t * [](start = 2, length = 15)

usually we would specify types as "const uint8_t* data()". Star and Apostrophe is right after type. Is fb convention required different syntax?

Reply:
Yes, Facebook's convention is to put the asterisk next to the variable name, and this convention is enforced through clang format.

---
In reply to: [320883747](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320883747) [](ancestors = 320883747)